react/components/Share/index.js

```diff
@@ -34,9 +45,11 @@ class Share extends Component {
     }
 
     return (
-      <div className="vtex-share">
+      <div className={loaderProps.classes
+        && classNames('vtex-share vtex-share-loader', loaderProps.classes.container)}>
```

Comment:
Don't you think that should be better to use destructure to avoid code repetition?